### ISSUE TITLE
Fix Spacing in Documentation/Papers page

### ISSUE THREAD

**Issue by Srinithyee** [ocaml/v2.ocaml.org#1376]
@patricoferris @gs0510 

I see that there is a lot of space in the left hand side of the page [here](https://ocaml.org/docs/papers.html#Industrial-Uses).

I'd like to fix the spacing from this:
![image](https://user-images.githubusercontent.com/34683520/113420321-bdbd6800-93e6-11eb-90e1-2baa065a6522.png)
![image](https://user-images.githubusercontent.com/34683520/113420345-c44bdf80-93e6-11eb-879a-b09664eba6f2.png)

to this:
![after1](https://user-images.githubusercontent.com/34683520/113420294-af6f4c00-93e6-11eb-9398-9b65aad33efa.png)
![after2](https://user-images.githubusercontent.com/34683520/113420299-b1390f80-93e6-11eb-9bd1-02f1b9089f5f.png)

If this looks like a good fix, I'd like to start working on it!

**Comment by pitag-ha** [comment]
Thanks a lot for the initiative! I do agree with you that with your approach the page would look a bit nicer, but introducing so many line breaks inside the breadcrumbs seems to make it harder to get an easy from and fast overview than it is now. Do you agree with that?

**Comment by Srinithyee** [comment]
@pitag-ha Yes, I do agree with that. Do you have any other suggestions to implement this? I thought of modifying the span class for the two sections of the page from span4 and span8 to span2 and span10 .Could you please let me know if that works?

**Comment by pitag-ha** [comment]
How would that look like?

**Comment by Srinithyee** [comment]
Like this
> ![after1](https://user-images.githubusercontent.com/34683520/113420294-af6f4c00-93e6-11eb-9398-9b65aad33efa.png)
> ![after2](https://user-images.githubusercontent.com/34683520/113420299-b1390f80-93e6-11eb-9bd1-02f1b9089f5f.png)

**Comment by pitag-ha** [comment]
Isn't that the same screenshot you already put into the issue description? I think it's harder to get an overview over the table of contents there than the way it is now.

**Comment by Srinithyee** [comment]
Oh Alright then, I think I'll probably close this issue , if you wouldn't like me to re space the page!

**Comment by gs0510** [comment]
@Srinithyee  I agree with @pitag-ha, I would also like the table of contents to be same as before, but the spacing solution is good. Do you want to provide a screenshot with those, and if it looks good you can go ahead? :)

**Comment by Srinithyee** [comment]
> @Srinithyee I agree with @pitag-ha, I would also like the table of contents to be same as before, but the spacing solution is good. Do you want to provide a screenshot with those, and if it looks good you can go ahead? :)

By spacing, I meant the space left on the left hand side of the page, on scrolling down. Like this one below:

![image](https://user-images.githubusercontent.com/34683520/113420345-c44bdf80-93e6-11eb-879a-b09664eba6f2.png)

Is this the spacing you'd like me to propose a solution to fix?

@gs0510 I've already contributed to 3 simple issues [#1441 ,#1440, #1369]. This does look simple to me. Would you suggest me taking this up or leaving it open for the others?

**Comment by gs0510** [comment]
@Srinithyee I'd leave it open for someone else to pick up! Thanks!

**Comment by Srinithyee** [comment]
@gs0510 Okay :)

**Comment by Srinithyee** [comment]
@kaniaganda Would you like to work on this as a good-first-issue, incase you haven't found one yet?

**Comment by kaniaganda** [comment]
@Srinithyee Yes! I would like to work on the issue. Thank you!

**Comment by kaniaganda** [comment]
@pitag-ha @gs0510 @Srinithyee Thank you for giving me a chance to work on the issue. After trying to work on it, I don't think I will be able to complete this issue for the time being. If there is anyone else who is interested in fixing the issue, please feel free to work on issue.

**Comment by sanjana-98** [comment]
Hi  @kaniaganda shall I work on this?

**Comment by gs0510** [comment]
@sanjana-98 Are you an Outreachy applicant? :) What project are you interested in? Since folks have been contributing for a few weeks, we already have quite a lot of contributions that we are considering for final applications. That being said we would anyway appreciate your contributions and hope you can participate in the next round :) Thanks!

**Comment by sanjana-98** [comment]
@gs0510 I am an outreachy applicant but couldn't contribute initially because of some personal issues. I would feel grateful if you could suggest some issues as am looking to contribute just for my own learning experience
